apidef/oas/upstream.go

```diff
@@ -30,16 +35,73 @@ func (u *Upstream) ExtractTo(api *apidef.APIDefinition) {
 }
 
 type ServiceDiscovery struct {
+	// Enabled enables the Service Discovery.
+	// Old API Definition: `service_discovery.use_discovery_service`
 	Enabled             bool   `bson:"enabled" json:"enabled"` // required
-	EndpointReturnsList bool   `bson:"endpointReturnsList,omitempty" json:"endpointReturnsList,omitempty"`
-	CacheTimeout        int64  `bson:"cacheTimeout,omitempty" json:"cacheTimeout,omitempty"`
-	ParentDataPath      string `bson:"parentDataPath,omitempty" json:"parentDataPath,omitempty"`
+	// QueryEndpoint is the endpoint to call, this would probably be Consul, etcd or Eureka K/V store.
```

Comment:
this would probably be Consul, etcd or Eureka K/V store.
Maybe change to:
this would commonly be Consul, etcd or Eureka K/V store.